<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SmartPaymentDialog</class>
 <widget class="QDialog" name="SmartPaymentDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Receive Payment</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Receipt No.</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Tenant Details:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="3">
      <widget class="QComboBox" name="cboUnit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Paid Via:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="cboPaidVia">
       <item>
        <property name="text">
         <string>Cash</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cheque</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Mobile Money</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Credit Card</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Other</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Amount Paid:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Select Unit:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2" colspan="2">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Cheque / Payment No.:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QLineEdit" name="txtChequeNo"/>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QLineEdit" name="txtReceiptNo"/>
     </item>
     <item row="3" column="3">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Receipt Date:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QDateEdit" name="dtpReceiptDate">
       <property name="displayFormat">
        <string>dd MMM yyyy</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="3">
      <widget class="QPlainTextEdit" name="txtTenantDetails">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>88</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="3" column="5">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="cmdAutoAllocate">
       <property name="text">
        <string>Allocate</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QDoubleSpinBox" name="spAmountReceived">
       <property name="maximum">
        <double>999999999.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Invoice Allocation:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTreeWidget" name="trvPaymentAllocation">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>5</verstretch>
      </sizepolicy>
     </property>
     <column>
      <property name="text">
       <string>Invoice No.</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Invoice Amount</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Paid</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Current Balance</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Allocation</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>New Balance</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="3">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Amount Received:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLineEdit" name="txtReceived"/>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Amount Allocated:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4" rowspan="2">
      <widget class="QLineEdit" name="txtAllocated"/>
     </item>
     <item row="0" column="0">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="1">
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="4">
      <widget class="QLineEdit" name="txtBalance"/>
     </item>
     <item row="3" column="3">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Balance:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="cmdSave">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
